docs/search/get-started-with-search/search-page/event-analytics.md

```diff
@@ -0,0 +1,37 @@
+---
+id: event-analytics
```

Comment:
we should a section describing what event sources are supported. Right now we only support deploy events from k8s events data collected using https://help.sumologic.com/docs/observability/kubernetes/